apps/src/weblab/WebLab.js

```diff
@@ -135,6 +142,18 @@ WebLab.prototype.init = function (config) {
     isProjectLevel: !!config.level.isProjectLevel,
   });
 
+  function onAddFileHTML() {
```

Comment:
Yes, was following convention in this file. Will make a note to switch this around in the future.